### DISCUSSION TITLE
Can Gems be made to be as performant as Gel?

### DISCUSSION THREAD

**Discussion by ylluminate** [ruby/rubygems#7875] [Ideas]
Gel is accomplishing some impressive things in moving beyond Ruby Gems' performance: https://github.com/gel-rb/gel#readme

Is there any chance exploration has been done to work on improving performance to bring it on par or to exceed Gel's achievements?

**Comment by simi** [comment]
Can you provide some numbers to compare? Share some Gemfile you're having speed issues, ... This kind of request is not really useful IMHO.

**Reply by ylluminate** [reply]
This is a discussion more than a request, you might want to look into: https://github.com/gel-rb/gel?tab=readme-ov-file#why-should-i-use-gel

**Reply by simi** [reply]
I'm aware of this, but this section is out of date. Bundler meanwhile adopted https://github.com/jhawthorn/pub_grub as well.

**Comment by indirect** [comment]
We have shipped several performance improvements in the last two years, so I think the answer to your question is "yes". Did you have any other questions?

**Comment by deivid-rodriguez** [comment]
Closing as resolved, since @indirect clearly answered the question.

**Reply by simi** [reply]
Btw. it would be good to ping friends at https://github.com/gel-rb/gel?tab=readme-ov-file#why-should-i-use-gel to redo the benchmark with latest RubyGems/Bundler.
